doc/source/starting.rst

```diff
@@ -0,0 +1,112 @@
+Getting Started
+===============
+
+Installation
+-----------
+The Distributions package is available through the Julia package system by running ``Pkg.add("Distributions")``. Throughout, we assume that you have installed the package.
+
+Starting With a Normal Distribution
+-----------
+
+We start by drawing :math:`100` observations from a standard-normal random variable. After setting up the environment through
+
+.. code-block:: julia
+
+    julia> using Distributions
+    julia> srand(123) # Setting the seed
+
+We first create a ``normal`` distribution and then obtain samples using ``rand``:
+
+.. code-block:: julia
+
+    julia> normal = Normal()
+    Normal(μ=0.0, σ=1.0)
+
+    julia> x = rand(normal, 100)
+    100-element Array{Float64,1}:
+      0.376264
+     -0.405272
+     ...
+
+You can easily obtain a pdf, cdf, percentile, and many other functions for a distribution. For instance, the median (50th percentile) and the 95th percentile for the standard-normal distribution are
+
+.. code-block:: julia
+
+    julia> quantile(Normal(), [0.5, 0.95])
+    2-element Array{Float64,1}:
+     0.0
+     1.64485
+
+The normal distribution is parameterized by a mean (location) and standard deviation (scale). To draw random samples from a normal distribution with mean 1 and standard deviation 2:
```

Comment:
I'd avoid mentioning location and scale.

I'd add a second clause in the second sentence to make it a complete sentence, e.g. "standard deviation 2, you can do something like:"